hydra/_internal/core_plugins/basic_sweeper.py

```diff
@@ -101,12 +104,13 @@ def split_arguments(
     ) -> List[List[List[str]]]:
 
         lists = []
+        merged_overrides = OrderedDict()
```

Comment:
I think the word "merged" here could be confusing (as no call is being made to `OmegaConf.merge`).
Maybe use "combined" instead?

Reply:
yes `merged` is probably not the best here - maybe just `final_overrides`? since we are neither merging nor combining the overrides here...